src/components/HeaderBar.js

```diff
@@ -2,17 +2,28 @@ import React from 'react'
 import '../assets/styles/headerBar.scss'
 import { Link } from 'react-router-dom'
 import { ROUTES } from '../routes'
+import { makeSelectToken } from '../store/selectors'
+import { useSelector } from 'react-redux'
 
 const HeaderBar = () => {
+  const token = useSelector(makeSelectToken())
+
```

Comment:
You can use conditional rendering here (but if only visibility needs to be changed, this approach is fine - you can just fix this typo to `authenticated`, instead of `authetnticated`).